templates/app/views/spree/checkout/_checkout_step.html.erb

```diff
@@ -0,0 +1,20 @@
+<%
+  form_url = update_checkout_path(@order.state)
```

Comment:
Why these variables? Can't we just have them in the code below, I can't see any advantage of having them here but maybe I'm missing something.

Reply:
Sure, I can inline them. TODO.